src/alchemlyb/workflows/abfe.py

```diff
@@ -307,7 +329,7 @@ def update_units(self, units=None):
             logger.info(f"Set unit to {units}.")
             self.units = units or None
 
-    def preprocess(self, skiptime=0, uncorr="dE", threshold=50):
+    def preprocess(self, skiptime=0, uncorr="dE", threshold=50, n_jobs=-1):
```

Comment:
see above

*    Is -1 safe as new default?
*    add docs (explanation)
*    add versionchanged